src/main/java/io/seqera/tower/cli/responses/ComputeEnvs/ComputeEnvsPrimarySet.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2021, Seqera Labs.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * This Source Code Form is "Incompatible With Secondary Licenses", as
+ * defined by the Mozilla Public License, v. 2.0.
+ */
+
+package io.seqera.tower.cli.responses.ComputeEnvs;
+
+import io.seqera.tower.cli.responses.Response;
+import io.seqera.tower.model.ComputeEnv;
+
+public class ComputeEnvsPrimarySet extends Response {
+
+    public final String workspaceRef;
+    public final ComputeEnv computeEnv;
+
+    public ComputeEnvsPrimarySet(String workspaceRef, ComputeEnv computeEnv) {
```

Comment:
When using the option `--json` the whole `computeEnv` object is show. I think that it would be better to here only pass the `computeEnv.id` and `computeEnv.name` so the `json` output is less verbose.

Reply:
I have added the following short CE version, let me know if it is to much:

```
{
  "credentials_id" : "G0lOxITHSVeAXhoKOU9uR",
  "name" : "AWS_Batch_eu-west-1",
  "id" : "1voNN3Mve8r49Ek6hFCN5v",
  "platform" : "aws-batch"
}
```